dev/Interop/StoragePickers/Microsoft.Windows.Storage.Pickers.idl

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) Microsoft Corporation and Contributors.
+// Licensed under the MIT License.
+
+#include <TerminalVelocityFeatures-StoragePickers.h>
+namespace Microsoft.Windows.Storage.Pickers
+{
+    [feature(Feature_StoragePickers)]
```

Comment:
>[feature(Feature_StoragePickers)] [](http://example.com/codeflow?start=4&length=33)

(not blocking initial checkin:) Note: Before these APIs go stable and remove "[feature(...)]", these will need to have a "[contract(...)]" added.

Reply:
Thank you for your comment, we will adding it later before move to stable release.